src/containers/UserEdit.jsx

```diff
@@ -40,7 +40,7 @@ class UserEdit extends React.Component {
 
   async componentWillMount() {
     if (!this.props.authType) {
-      await this.props.mutations.editUser(null);
+      await this.props.mutations.editUser(this.props.data.currentUser);
```

Comment:
do we need to test if `this.props.data` is available?  It's possible the component mounts without data available until the network connection is returned.

Can we add a comment here (once we figure out the need).
This needs testing/QA for:
* signup screen
* texter editing themselves screen
* admin editing a texter screen.

I worry that putting this.props.data.currentUser will only work for the second case (and possibly break the others)
Will defer to @ibrand on what we can QA vs. ensure we have a test for, for these use cases.

Reply:
We can have here a function that returns either `this.props.data.currentUser` or null, if it catches an error reading `this.props.data`. What do you think? @schuyler1d

Reply:
Ok, I will wait the QA to change it if needed. I was thinking about a `try/catch` block to define what `editUser` will be called with, but that should work too. And if that might be the case (this.props.data being undefined for some cases) I think it's a safe measure. I honestly don't know a good strategy to test it, specially if it is a race condition. Maybe changing the lifecycle method to `componentDidMount` could help?

Reply:
I think before QA, the logic I mentioned above needs to be in -- or are you arguing that we can leave it just as `props.data.currentUser`? even in the `!props.userId` case?

Reply:
Oh, my bad! Sorry. I will update it later and do some tests. Thanks!!!

Reply:
Hey @schuyler1d! Just tested the scenario you listed above and you were right. Editing a user in the list would bring the current logged in user info to Edit Page, not the user I clicked in. I reverted my changes and seems it is working now. Before, when I click to edit a user, the Edit Page was empty, but now it is bringing all the correct info. I think there was another change in the code that fixed that behavior I had seen.